Pinta.Core/Classes/BaseTool.cs

```diff
@@ -303,6 +303,14 @@ protected virtual void OnSaveSettings (ISettingsService settings)
 			settings.PutSetting (ANTIALIAS_SETTING, antialiasing_button.SelectedIndex);
 	}
 
+	/// <summary>
+	/// Called when the tool is deselected from the toolbox.
+	/// </summary>
+	protected virtual void OnChangeAntialias (object? sender, EventArgs e)
```

Comment:
To match some of the other methods' naming, I'd suggest `OnAntaliasingChanged`, and I'd also remove the `sender` / `e` arguments since they aren't useful for any of the derived classes that might override this? 
Also, it looks like the documentation comment might have been copied from elsewhere?

Reply:
Oops. Fixed both the documentation and naming convention, thanks.